.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,76 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master, dev-build ]
+
+env:
+  ORG: timescale # timescaledev
+  
+jobs:
+
+  timescaledb:
+
+    name: PG${{ matrix.pg }}${{ matrix.oss }}
+    runs-on: ubuntu-latest
+    strategy: 
+      matrix:
+        pg: [ 9.6, 10, 11, 12 ]
+        oss: [ "", "-oss" ]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: crazy-max/ghaction-docker-buildx@v2
+      with:
+        buildx-version: latest
+        skip-cache: false
+        qemu-version: latest
+
+    - name: Available platforms
+      run: echo ${{ steps.buildx.outputs.platforms }}
+
+    - name: Login to DockerHub Registry
+      run: echo ${{ secrets.DOCKERHUB_TOKEN }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
+
+    - name: Build and push multi-platform Docker image for TimescaleDB
+      run: make multi${{ matrix.oss }} ORG=$ORG PG_VER=pg${{ matrix.pg }}
+
+  timescaledb-bitnami:
```

Comment:
There are several reasons:
- It is good to discuss and decide if we want to build multi-arch for these.
- Nobody has investigated if it works to build 32bit for PostGIS.
- Main TimescaleDB images are built for OSS. We don't do this for PostGIS and bitnami. I guess this needs to be discussed.
- PostGIS needs TimescaleDB images, while bitnami depends only on PG. So they are not in the same build.

In general, I think we need to discuss how we deal with docker images. There are many questions to investigate and decide. This PR is step, which simplifies the discussion, since I guess some images were not considered due to manual builds.

Reply:
This is exactly what I think needs to be explained in comments. As a reviewer or someone looking at this code/config later it is not possible to know what you've learned and the reasons for certain decisions. That's why it is good to write it down.

I am not saying we need to resolve all issues right now, and a partial solution is good enough. But the open questions and reasons for going one way or the other should be documented.

Reply:
> This is exactly what I think needs to be explained in comments.

@erimatnor 
As you requested I already added comments to each build. So I don't follow what is your request for changes now.

Reply:
I added it, but I don't think it is correct approach to put so many details into the workflow code. There is an issue in the repo, which already tracks it.